gradle/tests/scan.bash

```diff
@@ -0,0 +1,10 @@
+  echo "buildScan {" >> "/hab/cache/src/hello-gradle/build.gradle"
```

Comment:
Why is this echoing to a file? Just include `build.gradle` file in the tests directory.